app/lib/common/themes/app_theme.dart

```diff
@@ -39,257 +40,32 @@ class EmojiConfig {
-          elevation: 0,
-          foregroundColor: Colors.white,
-          shape: RoundedRectangleBorder(
-            borderRadius: BorderRadius.circular(12),
-          ),
-        ),
-      ),
-      outlinedButtonTheme: OutlinedButtonThemeData(
-        style: OutlinedButton.styleFrom(
-          padding: const EdgeInsets.all(18),
-          shape: RoundedRectangleBorder(
-            side: BorderSide(
-              color: brandColorScheme.secondary,
-              width: 0.5,
-            ),
-            borderRadius: BorderRadius.circular(12),
-          ),
-          foregroundColor: Colors.white,
-          backgroundColor: Colors.transparent,
-        ),
-      ),
-      dividerTheme: const DividerThemeData(
-        indent: 75,
-        endIndent: 15,
-        thickness: 0.5,
-      ),
-      inputDecorationTheme: InputDecorationTheme(
-        border: OutlineInputBorder(
-          borderRadius: BorderRadius.circular(12),
-          borderSide: const BorderSide(color: Colors.white),
-        ),
-        focusedBorder: OutlineInputBorder(
-          borderRadius: BorderRadius.circular(12),
-          borderSide: const BorderSide(
-            color: Color(0xFF3AE3E0),
-          ),
-        ),
-        hintStyle: const TextStyle(
-          color: Color(0xFF898989),
-          fontWeight: FontWeight.w300,
-          fontSize: 14,
-        ),
-      ),
-      bottomNavigationBarTheme: BottomNavigationBarThemeData(
-        backgroundColor: brandColorScheme.navBarBackground,
-        unselectedLabelStyle: const TextStyle(
-          color: Colors.white,
-          fontSize: 12,
-          fontWeight: FontWeight.w400,
-        ),
-        selectedItemColor: Colors.white,
-        unselectedItemColor: Colors.white,
-        selectedLabelStyle: const TextStyle(
-          color: Colors.white,
-          fontSize: 12,
-          fontWeight: FontWeight.w400,
-          height: 1.5,
-        ),
-        selectedIconTheme: const IconThemeData(color: Colors.white, size: 18),
-        unselectedIconTheme: const IconThemeData(color: Colors.white, size: 18),
-        type: BottomNavigationBarType.fixed,
-        elevation: 0,
-      ),
-      navigationRailTheme: NavigationRailThemeData(
-        backgroundColor: brandColorScheme.navBarBackground,
-        indicatorColor: const Color(0xff1E4E7B),
-        unselectedLabelTextStyle: const TextStyle(
-          color: Colors.white,
-          fontSize: 12,
-          fontWeight: FontWeight.w400,
-        ),
-        selectedLabelTextStyle: const TextStyle(
-          color: Colors.white,
-          fontSize: 12,
-          fontWeight: FontWeight.w400,
-        ),
-        selectedIconTheme: const IconThemeData(color: Colors.white, size: 18),
-        unselectedIconTheme: const IconThemeData(color: Colors.white, size: 18),
-      ),
-    );
-  }
+  Color get tasksBG => primary;
+  Color get tasksFG => primaryContainer;
+  Color get taskOverdueBG => errorContainer;
```

Comment:
Where is `errorContainer` color declaration? Are we considering default `ThemeData errorContainer` color?

Reply:
yes, these all directly use the existing colorschemes now. See above for a more in-depth explanation.